.github/workflows/ci.yaml

```
@@ -10,87 +10,87 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  # Run unit tests.
+  # Run tests.
   test:
-    name: Test Suite
+    name: Tests
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Checkout sources
+      - name: Checkout repository
         uses: actions/checkout@v4
 
-      - name: Install stable toolchain
+      - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@stable
 
-      - name: Cache target directory
-        uses: Leafwing-Studios/cargo-cache@v1
-
-      - name: Install Dependencies
+      - name: Install dependencies
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
 
-      - name: Run cargo test
-        run: cargo test --all-targets
+      - name: Populate target directory from cache
+        uses: Leafwing-Studios/cargo-cache@v1
+
+      - name: Run tests
+        run: |
+          cargo test --workspace --all-targets --all-features
+          # TODO: Workaround for https://github.com/rust-lang/cargo/issues/6669
+          cargo test --workspace --docs --all-features
```

```suggestion
          # Workaround for https://github.com/rust-lang/cargo/issues/6669
          cargo test --workspace --docs
```
This reads as if the user needs to do something

---

Yeah fair enough. This is how I mark it in my own projects so I know to remove the workaround when it's no longer necessary.